roles/openshift_preflight/common/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+# check content available on all hosts
+- when:
+    - not containerized
+  block:
+
+    - name: determine if yum update will work
+      action: check_yum_update
+      register: r
+
+    - set_fact:
+        oo_preflight_check_results: "{{ oo_preflight_check_results + [r|combine({'_task': 'check content available on all hosts'})] }}"
```

Comment:
@rhcarvalho In general I agree, however it didn't work in the past with Ansible.

That said, the Ansible Core team has said that if it works now, then it should be supported in the future. So I don't see an issue with using that pattern here.